en/flight_modes/altitude.md

```diff
@@ -0,0 +1,55 @@
+# ALTITUDE CONTROL Flight Mode
+
+The ALTITUDE CONTROL flight mode maintains constant altitude when all remote control inputs are centered (i.e. no roll, pitch, yaw and at 50% throttle). The mode does not use GPS, and hence will not attempt to hold position against wind.

```

Comment:
Ah, didn't think about FW. Perhaps the more generic statement would be that in this mode the flight controller controls the altitude (rather than just _maintaining constant_ altitude) of the vehicle, with setpoints/changes commanded with the remote control.